README.md

```diff
@@ -56,6 +57,11 @@ type Settings struct {
   If `ReadyToTrip` is `nil`, default `ReadyToTrip` is used.
   Default `ReadyToTrip` returns true when the number of consecutive failures is more than 5.
 
+- `RecordError` is called when a non-nil error is returned by the function, and intends to differentiate errors that should be used for tripping the circuit or not.
+  If `RecordError` returns true, `CircuitBreaker` will not consider the operation as successful
+	If `RecordError` is `nil`, default `RecordError` is used.
+  Default `RecordError` returns true
```

Comment:
Can you clarify that if returns false, then the operation will be unsuccessful ?

Reply:
Default `RecordError` considers all errors should be recorded

☝️ Do you think that is clear?

Reply:
clarifies the logic behind `defaultRecordError `  but this text defines the RecordError semantic, so I'd suggest being explicit. 

With that phrace you will be saying the same as in line 61: the behaviour for `true` and there's not much said about the behaviour for `false`

Reply:
please re review